tests/test_pydantic.py

```diff
@@ -0,0 +1,18 @@
+from pydantic import BaseModel
+import pytest
+
+from s3_file_field.pydantic import S3FileFieldValue
+
+
+class Model(BaseModel):
```

Comment:
I would like to have a test to ensure that this is usable to save directly to the database as a Django mode.